### PR TITLE
Update JVM document to fix issue 13561

### PR DESCRIPTION
**Pull request by RobertShan2000**
Fix document issue on https://github.com/elastic/logstash/issues/13561

### CODE REVIEW

**Comment by logstashmachine** [comment]
Since this is a community submitted pull request, a Jenkins build has not been kicked off automatically. Can an Elastic organization member please verify the contents of this patch and then kick off a build manually?

File: docs/static/jvm.asciidoc [hunk 1]
```diff
@@ -50,7 +50,8 @@ Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.1+13-LTS, mixed mode)
 ==== `LS_JAVA_HOME`
 
 {ls} uses the Java version set in `LS_JAVA_HOME`. The `LS_JAVA_HOME` environment
-variable must be set for {ls} to operate correctly.
+variable must be set for {ls} to operate correctly if you prefer to use custom JDK
```
Comment (andsel) [inline docs/static/jvm.asciidoc:53]: I would rewrite as:
```
If you prefer to use custom JDK other than the version that is bundled, then the `LS_JAVA_HOME` environment
variable must be set.
```

Comment (andsel) [reply docs/static/jvm.asciidoc:53]: cc @karenzone

**Review by andsel — CHANGES_REQUESTED** [review]
Suggested a reworking, to make the `must` sound less imperative.
@karenzone  WDYT?

@RobertShan2000 please, sign the CLA so that your contribution could be accepted.

**Comment by RobertShan2000** [comment]
> Suggested a reworking, to make the `must` sound less imperative. @karenzone WDYT?
> 
> @RobertShan2000 please, sign the CLA so that your contribution could be accepted.

@andsel I sign the CLA, would the CLA bot check again within the PR?

File: docs/static/jvm.asciidoc (hunk 1 above)
Comment (RobertShan2000) [reply docs/static/jvm.asciidoc]: @andsel I updated the sentence as your suggestion

Comment (andsel) [reply docs/static/jvm.asciidoc]: I would ask  @karenzone for a quick eye

**Review by andsel — APPROVED** [review]

File: docs/static/jvm.asciidoc [hunk 2]
```diff
@@ -49,7 +49,8 @@ Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.1+13-LTS, mixed mode)
 [[java-home]]
 ==== `LS_JAVA_HOME`
 
-{ls} uses the Java version set in `LS_JAVA_HOME`. The `LS_JAVA_HOME` environment
+{ls} uses the Java version set in `LS_JAVA_HOME`. If you prefer to use custom 
+JDK other than the version that is bundled, then the `LS_JAVA_HOME` environment
 variable must be set for {ls} to operate correctly.
```
Comment (karenzone) [inline docs/static/jvm.asciidoc:54]:
```suggestion
Logstash includes a bundled JDK which has been verified to work with each
specific version of Logstash, and generally provides the best performance and
reliability. 
If you need to use a JDK other than the bundled version, then set the
`LS_JAVA_HOME` environment variable to the version you want to use.
```

Comment (karenzone) [reply docs/static/jvm.asciidoc:54]: Thank you for making this fix, @RobertShan2000!  This issue definitely needed clarification. Thank you for making it happen. 
Please consider updating the existing first sentence to emphasize the benefits of sticking to the bundled version a bit more (as indicated in the suggestion). We want people to have the option to set `LS_JAVA_HOME`, but we recommend that people stick to the bundled JDK unless they have a compelling reason for using another one.

**Review by karenzone — COMMENTED** [review]
Thank you for adopting this issue and making the fix, @RobertShan2000. I left a suggestion for including additional information inline for your consideration.

File: docs/static/jvm.asciidoc (hunk 2 above)
Comment (RobertShan2000) [reply docs/static/jvm.asciidoc]: > 

@karenzone I refine the sentence as your suggestion, please let me know if you have any questions.

**Review by andsel — APPROVED** [review]
LGTM

**Comment by andsel** [comment]
So many thanks @RobertShan2000 for your welcome contribution! :tada:

**Comment by RobertShan2000** [comment]
> So many thanks @RobertShan2000 for your welcome contribution! 🎉

My pleasure to contribute this.

**Pull request merged**